test

```diff
@@ -21,3 +21,4 @@ test coroutines-co.js
 test coroutines-bluebird.js
 test await.compiled.js
 test futures.js
+test promises2.js
```

Comment:
I think __promises-ramda.js__ is a good name for the file. The most significant difference between your version and __promises.js__ is the use of `R.invoker(1, 'then')` to solve the composition problem.